### ISSUE TITLE
Abnormal circulating Tenascin concentration

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6112]
**Preferred term label:** Abnormal circulating Tenascin concentration

**Synonyms**

**Definition (free text, please give PubMed ID)** Concentration of tenascin in the blood is outside the normal range., PMID: 25793574

**Parent term (use hpo.jax.org/app)** Abnormal circulating protein level HP:0010876

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:** Decreased circulating Tenascin concentration

**Synonyms**

**Definition (free text, please give PubMed ID)** Concentration of tenascin in the blood is below the normal range.

**Parent term (use hpo.jax.org/app)** Abnormal circulating Tenascin concentration

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:** Increased circulating Tenascin concentration

**Synonyms**

**Definition (free text, please give PubMed ID)** Concentration of tenascin in the blood is above the normal range.

**Parent term (use hpo.jax.org/app)** Abnormal circulating Tenascin concentration

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6238343/

**Comment by pnrobinson** [comment]
Added a term for 'elevated'
